src/js/structs/NodesList.d.ts

```diff
@@ -0,0 +1,10 @@
+import List from "./List";
+import Node from "./Node";
+
+export default class NodesList extends List {
+  constructor(props: any);
+  filter: any;
+  sumUsedResources: any;
+  sumResources: any;
+  getItems(): Array<Node>;
```

Comment:
I wonder why tslint does not complaint here, for simple types it should be `Node[]`